src/main/java/core/basesyntax/figures/RightTriangle.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.figures;
+
+import java.util.Locale;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+    private double area;
```

Comment:
The variable 'area' should not be a class level field. Instead, 'area' can be calculated on the fly when 'getArea()' is called, or it can be stored as a local variable within 'calculateArea()' method if needed.